frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div>
+    <div class="container-fluid" data-feedback-code-mirror>
+      <textarea v-show="false" ref="cm-editor" v-model="value"></textarea>
+    </div>
+    <div class="container-fluid text-right py-2">
+      <button
+        class="btn btn-primary mx-2"
+        :disabled="!numberOfComments"
+        @click.prevent="saveFeedbackList"
+      >
+        {{ T.submissionSendFeedback }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Ref } from 'vue-property-decorator';
+import T from '../../lang';
+import CodeMirror from 'codemirror';
+import { EditorOptions, languageModeMap, modeList } from './CodeView.vue';
+import Feedback, { ArenaCourseFeedback, FeedbackStatus } from './Feedback.vue';
+
+for (const mode of modeList) {
+  require(`codemirror/mode/${mode}/${mode}.js`);
+}
+
+@Component({
+  components: {},
+})
+export default class FeedbackCodeView extends Vue {
+  @Prop() language!: string;
+  @Prop() value!: string;
+  @Prop({ default: () => new Map() }) feedbackList!: Map<
+    number,
+    ArenaCourseFeedback
+  >;
+  @Ref('cm-editor') private readonly cmEditor!: HTMLTextAreaElement;
+
+  T = T;
+  mode = languageModeMap[this.language] ?? languageModeMap['cpp17-gcc'];
+  numberOfComments = 0;
+
+  get feedbackListEmpty(): number {
+    return this.feedbackList.size;
+  }
```

Comment:
```suggestion
  get isFeedbackListEmpty(): boolean {
    return this.feedbackList.size === 0;
  }
```

también, esto parece que no se está usando.

Reply:
En algún punto dejé de usarlo y olvidé quitarlo, supongo. Así que se va